### DISCUSSION TITLE
have 2 versions of reanimated installed on same project

### DISCUSSION THREAD

**Discussion by Learto** [software-mansion/react-native-reanimated#2060] [Q&A]
I have a big project with `"react-native-reanimated": "^1.3.0"` already installed. I want to install the react-native reanimated v2.

I found this solution : https://stackoverflow.com/questions/26414587/how-to-install-multiple-versions-of-package-using-npm :

```
yarn add react-native-reanimated-next@npm:react-native-reanimated@next
```

The rest of the installation is the same than a normal installation ?

For example, this should be :

```js
  module.exports = {
      ...
      plugins: [
          ...
          'react-native-reanimated/plugin',
      ],
  };
```
or 

```js
  module.exports = {
      ...
      plugins: [
          ...
          'react-native-reanimated-next/plugin',
      ],
  };
```

Thanks for answers.

**Comment by jgonet** [comment]
You can't use two instances of Reanimated since we can only have one native module running. Trying to use them will lead to duplicated registration on the native side and your application will crash.

We tried to make the transition from v1 as easy as it could be - Reanimated v2 includes v1 stuff and we only require renaming interpolate and Easing. Please refer to [this doc page](https://docs.swmansion.com/react-native-reanimated/docs/migration) for more information.

**Reply by Learto** [reply]
Thanks for answer! So I don't need to make the `Babel plugin`, `Android` and `IOS` part of the [installation page](https://docs.swmansion.com/react-native-reanimated/docs/installation/) ?

**Reply by jgonet** [reply]
No, you still need to do those parts. The "easy" part applies to migrating your codebase to v2 😄
